src/workspacer.Shared/Workspace/IWorkspace.cs

```diff
@@ -128,6 +130,8 @@ public interface IWorkspace
         /// </summary>
         void DoLayout();
 
+        event AfterWorkspaceLayoutDelegate AfterLayout;
```

Comment:
Event-names should typically be given a `On`-prefix.

Reply:
`OnLayoutCompleted` sounds good to me, so it has been changed accordingly.

Reply:
Not seeing the change here though. Have you remembered to push? :smile:

Reply:
I used a force push to make sure that non-pr code doesn't interfere with future branches that make those changes, so it doesn't detect the change :). If you look at the changes it's definitely `OnLayoutCompleted` now